### PR TITLE
DAOS-7735 test: Added progress indicator to autotest

### PR DESCRIPTION
**Pull request by chrisphoffman**
Added progress indicator to autotest

Signed-off-by: Christopher Hoffman <christopherx.hoffman@intel.com>

### CODE REVIEW

File: src/utils/daos_autotest.c [hunk 1]
```diff
@@ -186,6 +222,9 @@ oS1(void)
 	new_oid();
 	daos_obj_generate_oid(coh, &oid, 0, OC_S1, 0, 0);
 
+	total_nr = 1000000;
+	setup_progress();
+
 	for (i = 0; i < 1000000; i++) {
```
Comment (bfaccini) [inline src/utils/daos_autotest.c:228]: should use "i < total_nr" now, right ?

File: src/utils/daos_autotest.c [hunk 2]
```diff
@@ -215,6 +256,9 @@ oSX(void)
 	new_oid();
 	daos_obj_generate_oid(coh, &oid, 0, OC_SX, 0, 0);
 
+	total_nr = 10000;
+	setup_progress();
+
 	for (i = 0; i < 10000; i++) {
```
Comment (bfaccini) [inline src/utils/daos_autotest.c:262]: same here

**Review by bfaccini — CHANGES_REQUESTED** [review]

File: src/utils/daos_autotest.c (hunk 1 above)
Comment (chrisphoffman) [reply src/utils/daos_autotest.c:228]: That's a good point. I did consider that but I was hesitant using total_nr here as that variable use is for the progress bar only. I also considered creating a new variable for the number of records. That way total_nr and the for loop would use that variable. If I had to choose the former option would be the cleanest. I can proceed with that.

**Review by chrisphoffman — COMMENTED** [review]
for-loop conditional change has been pushed.

**Comment by bfaccini** [comment]
too bad is that autotest has not been run under CI due to "Skipping until DAOS-7946 is fixed" reason !!...
may be you should use DAOS-7946 (also yours !), instead of DAOS-7735 (and may be you could also explicitly link both tickets ?), in your commit header to bypass the filter ??

**Review by bfaccini — DISMISSED** [review]

**Comment by chrisphoffman** [comment]
Can everyone take a look at this again please? Also all tests are reporting as completed, but github isn't showing it properly.

**Review by bfaccini — APPROVED** [review]

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Pull request merged**